tests/qa/run

```diff
@@ -73,17 +68,17 @@ run_test() {
 	fi
 
 	JSON=$(curl -sk \
-		-u "$QA_AUTOMATION_USERNAME":"$QA_AUTOMATION_PASS" \
-		-X POST -H "Content-Type: application/json" \
-		-d @$json_in \
-		https://qa-automation-center.redislabs.com/processes 2>&1)
+   		-X POST -H "Content-Type: application/json" \
+   		-H "Authorization: Bearer $QA_AUTOMATION_TOKEN" \
+   		-d @$json_in \
+   		https://opereto.qa.redislabs.com/processes 2>&1)
```

Comment:
Let's put ths URL in a variable and reuse it, so that the next time, we don't need to edit in two places.